src/tv2-common/__tests__/transition-from-string.spec.ts

```diff
@@ -1,18 +1,18 @@
-import { TSR } from 'blueprints-integration'
-import { AtemTransitionStyleFromString } from '../atemTransitionStyleFromString'
+import { TransitionStyle } from 'tv2-common'
+import { TransitionStyleFromString } from '../transitionStyleFromString'
 
 describe('Transition From String', () => {
 	it('Converts strings', () => {
```

Comment:
Maybe it is me who has misunderstood it. 
Is the 'unknown' case, meant to be any input, e.g 'horse', 'object', 'unknown' and etc.

If the 'unknown' case isn't for any input, then what about if an empty string is given?
Like the 'unknown' case, id expect a `TransitionStyle.CUT`, but at the moment there is no empty case.
Might not need to be handled, if `' '` is never a valid input.

Reply:
Technically it's impossible to have anything other than the four currently supported words passed to that function, so the 'unknown', which indeed corresponds to any string imaginable, will never get there in real life, and some other test cases are redundant as well. https://github.com/tv2/sofie-blueprints-inews/blob/3f8679497aeff1ce4791957ab59ea052bfe3d216/src/tv2-common/inewsConversion/converters/ParseBody.ts#L559